src/hooks/useENSProfileForm.ts

```diff
@@ -34,6 +44,7 @@ export default function useENSProfileForm({
   const [values, setValues] = useState(records);
 
   // Set initial records in redux depending on user input (defaultFields)
+  // TODO: this is not working as expected when going back and changing name
```

Comment:
when you're in the form, then go back, change the name and go to the form again the `values` object is not updated to the new `records` (because the name changed), it just works the first time in the screen because of 

```
const [values, setValues] = useState(records);
```

Reply:
I can fix this